src/m365/aad/commands/user/user-get.spec.ts

```diff
@@ -101,16 +134,45 @@ describe(commands.USER_GET, () => {
 
   it('retrieves user using user name', (done) => {
     sinon.stub(request, 'get').callsFake((opts) => {
-      if ((opts.url as string).indexOf('https://graph.microsoft.com/v1.0/users?$filter=userPrincipalName eq ') > -1) {
-        return Promise.resolve({ value: [{ "id": "68be84bf-a585-4776-80b3-30aa5207aa21", "businessPhones": ["+1 425 555 0100"], "displayName": "Aarif Sherzai", "givenName": "Aarif", "jobTitle": "Administrative", "mail": null, "mobilePhone": "+1 425 555 0100", "officeLocation": null, "preferredLanguage": null, "surname": "Sherzai", "userPrincipalName": "AarifS@contoso.onmicrosoft.com" }] });
+      if ((opts.url as string).indexOf(`https://graph.microsoft.com/v1.0/users?$filter=userPrincipalName eq '${encodeURIComponent(userName)}'`) > -1) {
+        return Promise.resolve({ value: [resultValue] });
       }
 
       return Promise.reject('Invalid request');
     });
 
-    command.action(logger, { options: { debug: false, userName: 'AarifS@contoso.onmicrosoft.com' } }, () => {
+    command.action(logger, { options: { debug: false, userName: userName } }, () => {
       try {
-        assert(loggerLogSpy.calledWith({ "id": "68be84bf-a585-4776-80b3-30aa5207aa21", "businessPhones": ["+1 425 555 0100"], "displayName": "Aarif Sherzai", "givenName": "Aarif", "jobTitle": "Administrative", "mail": null, "mobilePhone": "+1 425 555 0100", "officeLocation": null, "preferredLanguage": null, "surname": "Sherzai", "userPrincipalName": "AarifS@contoso.onmicrosoft.com" }));
+        assert(loggerLogSpy.calledWith(resultValue));
+        done();
+      }
+      catch (e) {
+        done(e);
+      }
+    });
+  });
+
+  it('retrieves user using @username token', (done) => {
```

Comment:
I suppose 
'...using @**me**username token'
will be more precise 👍, but now it is also quite clear 😋

Reply:
Changed it. Thank you